portal/db/postgresql/update-9.sql

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
The whitespace changes in this file are due to merge conflicts. Can this file be reverted to the original version so it doesn't show up as changed by this branch?